feature-group-definitions/scroll-snap.yml

```
@@ -1,4 +1,5 @@
 name: Scroll snap
+description: CSS scroll snap properties control the panning and scrolling behavior within a scroll container.
```

Since the properties aren't named:

```suggestion
description: CSS scroll snap controls the panning and scrolling behavior within a scroll container.
```

Also paging @flackr for review of this description.

---

Looks good to me, but leaving this one open for @flackr to comment.

---

I'll ping @flackr, but let's not block on that, I'd love to get this merged sooner rather than later, when @ddbeck has reviewed.

---

This text looks good to me!